docs/features/crosssections-features/searchtab.md

```diff
@@ -0,0 +1,5 @@
+# Search tab
+
+In order to filter any list of objects that is available in the application you can make use of the search tab feature. This tab is present in the top right part of the application and filters the objects from the current screen, by their names. 
```

Comment:
Mention that search can filter by resource name or labels. Multiple filters can be added, they are combined as logical OR.